plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -80,7 +80,7 @@ export const setupProjects = [
 
 export default defineConfig( {
 	timeout: 120 * 1000,
-	expect: { timeout: 20 * 1000 },
+	expect: { timeout: CI ? 20 * 1000 : 10 * 1000 },
```

Comment:
Is this change intentional?

Reply:
Yes, the env is much faster locally and should not need more than 10s. Even in CI 20s is too much I think, and I plan on tweaking that in the future. But that's for another time.
We'll need to override these for the external environments though.